website/source/docs/jobspec/servicediscovery.html.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: "docs"
+page_title: "Service Discovery in Nomad"
+sidebar_current: "docs-service-discovery"
+description: |-
+  Learn how to add service discovery to jobs
+  ---
+
+# Service Discovery
+
+Nomad enables dynamic scheduling on compute resources to run services at scale
+on compute nodes. Size of an application cluster varies depending
+on volume of traffic, health of services, etc thereby the network topology of
+services are constanly changing. This introduces the challenge of discovery of services, 
```

Comment:
I think the first two sentences are confusing. Also I wouldn't use `compute resources` and `compute nodes`. Instead tasks and Nomad Clients